extensions/dicom-pdf/src/index.js

```diff
@@ -1,11 +1,17 @@
-import asyncComponent from './asyncComponent.js';
+import React from 'react';
 import OHIFDicomPDFSopClassHandler from './OHIFDicomPDFSopClassHandler.js';
 
-const ConnectedOHIFDicomPDFViewer = asyncComponent(() =>
-  import(
-    /* webpackChunkName: "ConnectedOHIFDicomPDFViewer" */ './ConnectedOHIFDicomPDFViewer'
-  )
-);
+const Component = React.lazy(() => {
+  return import('./OHIFDicomPDFSopClassHandler');
```

Comment:
This doesn't seem right. That's not a component, it's just a function. You would need to put ConnectedOHIFDicomPDFViewer in here instead and rename the function on line 8, I suppose.

Reply:
Okay. I have updated the file with the correct component.

Reply:
Thanks, it's appears to be fixed now.